qsignups/database/__init__.py

```diff
@@ -77,13 +76,31 @@ def update_record(cls: T, id, fields):
       finally:
         session.commit()
         close_session(session)
+    
+    def update_records(cls: T, filters, fields):
+      session = get_session()
+      try:
+        session.query(cls).filter(and_(*filters)).update(fields, synchronize_session='fetch')
+        session.flush()
+      finally:
+        session.commit()
+        close_session(session)
 
-    def create_record(record: BaseClass) -> int:
+    def create_record(record: BaseClass) -> BaseClass:
+      session = get_session()
+      try:
+        session.add(record)
+        session.flush()
+        return record
+      finally:
+        session.commit()
+        close_session(session)
```

Comment:
@paul-christmann I changed this to return a record rather than the record id... based on how you implemented in [home.py](qsignups/slack/forms/home.py) I was guessing that's how you meant to do it.